08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <title>Blog</title>
+
+    <link rel="stylesheet" href="css/homework-style.css" type="text/css">
+
+</head>
+
+<body>
+
+    <header>
+
+        <h1>Recent From Blog</h1>
+
+        <p>Nullam viderer at has, usu an saperet ornatus nominavi. Enim aeque legendos ut mea, his rebum altera te. Ad mel soleat accusam appetere, ad has appareat interpretaris conclusionemque.</p>
+
+    </header>
+
+    <main class="clearfix">
+
+        <section class="main_box">
+
+                <img src="img/keyboard.jpg">
+
+                <h3>Ad mel soleat accusam appetere</h3>
+
+                <p>Ut usu agam quodsi inimicus. Fabellas definitiones id eam. Sea no oratio quaeque, labitur omittantur no pri.</p>
+
+                <a href="#">Read More</a>
+
+        </section>
+
+        <aside>
+
+            <div class="side_boxes">
+
+                <div>
+
+                    <img src="img/laptop.jpg" />
+
+                    <h3>Quis enim hinc an</h3>
+
+                    <p>Usu et minim detraxit, mutat sensibus vel ne, meliore dolorem id nam.</p>
+
+                </div>
+
+                <div id="side_middle_box">
+
+                    <img src="img/desk.jpg" />
+
+                    <h3>Eam inani suscipit</h3>
+
+                    <p>Ne vix illum movet, audiam voluptatibus te eos. Cu quod iudico consulatu pri.</p>
+
+                </div>
+
+                <div>
+
+                    <img src="img/window.jpg" />
+
+                    <h3>Quis enim hinc an</h3>
+
+                    <p>Usu et minim detraxit, mutat sensibus vel ne, meliore dolorem id nam.</p>
+
+                </div>
+
+            </div>
+
+        </aside>
+
+    </main>
+
```

Comment:
your footer is off-center here. and the UL's LIs are indented when they shouldn't. Also, on my screen the last box of the footer boxes goes to the next line.  the positioning is off for the elements. consider have the three main "boxes" being the same width.